templates/nomad.yaml

```diff
@@ -0,0 +1,37 @@
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: "nomad"
+  name: "nomad"
+spec:
+  ports:
+  - port: 8000
+    targetPort: 8000
+  selector:
+    app: "nomad"
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: "nomad"
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: "nomad"  
+  replicas: 1
```

Comment:
Because nomad is a core-service having only one replica means If there is more traffic there will be a significant impact or the app may crash while performing its tasks.